### PR TITLE
Waiter and small new test

### PR DESCRIPTION
**Pull request by 24alinka**
Hello. I added some new tests and added a waiter. I'm also trying to add tests to the new Chat page. But so far it hasn't come out. I have a problem with my tests on the "Start page", they worked before adding the Chat page, but now they don't. Could you please check what is the problem?

### CODE REVIEW

File: tests/test_chat_page.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    log = logging.getLogger("[ChatMessage]")
+
+    @pytest.fixture(scope="function")
+    def chat_page(self):
+        # Pre-conditions
+        driver = webdriver.Chrome(DRIVER_PATH)
+        driver.get(BASE_URL)
+        driver.implicitly_wait(1)
+        # Steps
+        yield TestChatMessage(driver)
```
Comment (QALightLecorPython) [inline tests/test_chat_page.py]: Don't know what you're trying to do, but this won't work. 
When you're getting the BASE_URL you're on StartPage, not ChatPage or any other page.
Chat by itself is not the Page it's an element like Header, so it should be implemented in same way.

**Review by QALightLecorPython — CHANGES_REQUESTED** [review]

**Comment by QALightLecorPython** [comment]
>  I have a problem with my tests on the "Start page", they worked before adding the Chat page, but now they don't. Could you please check what is the problem? 

I cannot check what is the problem. You could investigate it and describe it to me, so I can help you with the solution.

**Comment by 24alinka** [comment]
Hello. I fixed some tests. But I have issues with all my tests. I think it has to do with imports. Could you recheck and tell me what needs to be fixed?

File: pages/header.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from pages.base_page import BasePage
+from pages.hello_page import HelloPage
+
+
+class Header(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.constants = HeaderConsts()
+        self.hello_page = HelloPage(self.driver)
```
Comment (QALightLecorPython) [inline pages/header.py:19]: HelloPage is not a part of the header. Header is the part of the hello page ;)

File: pages/header.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    def navigate_to_create_post_page(self):
+        """Click on create post button"""
+        self.click(self.constants.CREATE_POST_BUTTON_XPATH)
+        from pages.create_post_page import CreatePostPage
+        return CreatePostPage(self.driver)
+
+    def sign_in(self, username, password):
```
Comment (QALightLecorPython) [inline pages/header.py:34]: This is actually related to the start page not to the header

File: pages/start_page.py [hunk 4]
```diff
@@ -2,80 +2,64 @@
...
+# from pages.hello_page import HelloPage
 
 
 class StartPage(BasePage):
 
     def __init__(self, driver):
         super().__init__(driver)
         self.constants = StartPageConstants()
+        self.header = Header(self.driver)
+        # self.hello_page = HelloPage(self.driver)
```
Comment (QALightLecorPython) [inline pages/start_page.py]: Hello page is not part of the start page. This is a different page

File: pages/header.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
+from constants.header import HeaderConsts
+from pages.base_page import BasePage
+from pages.hello_page import HelloPage
```
Comment (QALightLecorPython) [inline pages/header.py:6]: Do not use global imports for pages

**Review by QALightLecorPython — CHANGES_REQUESTED** [review]

**Pull request closed without merging**